packages/backend/src/models/NoteDraft.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * SPDX-FileCopyrightText: syuilo and misskey-project
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Entity, Index, JoinColumn, Column, PrimaryColumn, ManyToOne } from 'typeorm';
+import { noteVisibilities, noteReactionAcceptances } from '@/types.js';
+import { id } from './util/id.js';
+import { MiUser } from './User.js';
+import { MiChannel } from './Channel.js';
+import type { MiDriveFile } from './DriveFile.js';
+
+@Entity('note_draft')
+export class MiNoteDraft {
+	@PrimaryColumn(id())
+	public id: string;
+
+	@Index()
+	@Column({
+		...id(),
+		nullable: true,
+		comment: 'The ID of reply target.',
+	})
+	public replyId: MiNoteDraft['id'] | null;
+
+	@ManyToOne(type => MiNoteDraft, {
+		onDelete: 'CASCADE',
+	})
+	@JoinColumn()
+	public reply: MiNoteDraft | null;
```

Comment:
```suggestion
	@ManyToOne(type => MiNote, {
		onDelete: 'CASCADE',
	})
	@JoinColumn()
	public reply: MiNote | null;
```

こうじゃないかしら（renoteも同様 (わかってない)）

Reply:
確かにそう
なおします